### PR TITLE
yearn additional vaults

### PR DESCRIPTION
**Pull request by AyadiGithub**
## What?
Add yearn additional v2 vaults and registry

## How? 
I used [abi-parser](https://nansen-contract-parser-prod.web.app/) to build the table definitions

### CODE REVIEW

File: dags/resources/stages/parse/table_definitions/yearn/V2_VaultRegistry_event_NewVault.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "apiVersion",
+                "type": "STRING"
+            }
+        ],
+        "table_description": "",
+        "table_name": "V2_VaultRegistry_event_NewVault"
```
Comment (TimNooren) [inline dags/resources/stages/parse/table_definitions/yearn/V2_VaultRegistry_event_NewVault.json:74]: Should this be `V2VaultRegistry_event_NewVault`, following the naming convention of the other definitions?

**Review by TimNooren — CHANGES_REQUESTED** [review]

File: dags/resources/stages/parse/table_definitions/yearn/V2_VaultRegistry_event_NewVault.json (hunk 1 above)
Comment (AyadiGithub) [reply dags/resources/stages/parse/table_definitions/yearn/V2_VaultRegistry_event_NewVault.json]: Yes sir! Changed it

**Review by TimNooren — APPROVED** [review]

**Comment by TimNooren** [comment]
@AyadiGithub: Data should be available:)

**Comment by medvedev1088** [comment]
Since V2AdditionalVaults_event_Transfer was edited, it required a full rerun of that table. I just did that so the data now shows up in the table:

```
SELECT * 
FROM `blockchain-etl.ethereum_yearn.V2AdditionalVaults_event_Transfer` 
where contract_address = '0xa696a63cc78dffa1a63e9e50587c197387ff6c7e'
order by block_timestamp desc
```

cc @TimNooren @AyadiGithub

**Comment by TimNooren** [comment]
@medvedev1088 Thanks, I did not fully realise👍

**Pull request merged**